<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>270</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GBA Mus Ripper GUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="inputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ROM file path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="outputLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output directory path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="splitSoundBanks">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>120</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Split sound banks</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="giveGMNames">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>361</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Give instruments GM-compatible names</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="avoidCh10">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>150</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Don't use MIDI channel 10</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="outputRaw">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>180</y>
      <width>211</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output raw MIDIs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="makeXGCompliant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>XG-compliant MIDIs</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="manualAddressEnable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>176</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Manually specify address:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="manualAddress">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="outputPathEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>523</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseOutputButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>80</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="romPathEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>523</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseRomButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>630</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
